doc/source/buildoptions.rst

```diff
@@ -168,6 +168,8 @@ ready.
   access. Defaults to 5000.
 - ``--extra-manifest-xml``: Extra xml to write directly inside the
   ``<manifest>`` element of AndroidManifest.xml.
+- ``--extra-manifest-application-arguments``: Extra arguments to be
```

Comment:
Should this be called `--extra-manifest-application-attributes` instead? A thingy inside the angle brackets of an XML tag is an attribute, not an argument.

But then again ‘argument’ might be a manifest-specific term which is precisely correct to use in this circumstance; ignore me if so.

Reply:
I agree completely. However, the option already exists in p4a. All I did was wire it up into more bootstraps and add documentation for it. I think it would be reasonable to add the proper option upstream and deprecate the old one, but I don't want to do that downstream. I'll take a poke at that when I send this upstream.

Reply:
Oops, I missed that this option already exists in some parts of p4a. Your reasoning makes a lot of sense then.